templates/pipelines/common-app-ci.yaml

```diff
@@ -41,6 +41,13 @@ parameters:
   type: object
   default: 
     failOnThreshold: 'moderate'
+- name: sonarConfig
+  displayName: Details for Sonar Cloud Analysis
+  type: object
+  default: 
+    sonarConnection: 'DEFRA SonarCloud'
+    organization: defra
+    projectKeyPrefix: 'adp-'
```

Comment:
Because these are common templates we may not want the default to be ADP...
not sure what should be the default value.. @kbabigumira  @DaRoszko